templates/_helpers.yaml

```diff
@@ -53,15 +53,21 @@
 
 {{/* User defined Airflow environment variables */}}
 {{- define "custom_airflow_environment" }}
-  # Dynamically created environment variables
-  {{- range $i, $config := .Values.env }}
-  - name: {{ $config.name }}
-    value: {{ $config.value | quote }}
-    {{- if eq $.Values.executor "KubernetesExecutor" }}
-  - name: AIRFLOW__KUBERNETES_ENVIRONMENT_VARIABLES__{{ $config.name }}
-    value: {{ $config.value | quote }}
-    {{- end }}
+  {{- if .Values.mountAllFromSecretName }}
```

Comment:
@schnie Is there a way we can test the if condition with the secretName equals {{ .Release.Name }}-env inside the if condition? I tried that, but helm doesnt like it.

Reply:
I think this is fine. Houston will take care of passing the right format in. Should be able to trust that. Plus it leaves the chart "generic", so someone else could put whatever they want there if they pick it up for AC purposes.